src/app.css

```diff
@@ -150,3 +150,12 @@ p {
         --drawer-padding: 0.75rem 1.25rem;
     }
 }
+
+.screen-reader-only {
+    position: absolute;
+    left: -10000px;
+    top: auto;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
```

Comment:
Nice! I'm not a hundred percent sure about this, but I think there is a 'more recent' way to do this, which covers some issues the 'position: absolute' method uses. It's covered in [this CSS-tricks post](https://css-tricks.com/inclusively-hidden/) (taken from [Inclusively Hidden](https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html) by Scott O'Hara)) among other things. 

```css
.screen-reader-only:not(:focus):not(:active) {
  clip: rect(0 0 0 0); 
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap; 
  width: 1px;
}
```

I've personally used a variant of that, which looks like this

```css
    border: 0;
    clip: rect(0 0 0 0);
    height: auto;
    margin: 0;
    overflow: hidden;
    padding: 0;
    position: absolute;
    width: 1px;
    white-space: nowrap;
```

Scott's equivalent would be (and I'd probably recommend you use this one over mine): 

```suggestion
    clip: rect(0 0 0 0); 
    clip-path: inset(50%);
    height: 1px;
    overflow: hidden;
    position: absolute;
    white-space: nowrap; 
    width: 1px;
```

I'll let you decide what the right way forward here is (you may know more about this than me!), but I'd generally trust Chis Coyier and Scott O'Hara on these things. The `:not(:focus):not(:active)` seems to be to allow the text to become visible when the element is focused (like with skip links).

Reply:
More content:

[WebAIM seems to say absolute positioning and clipping do the same thing](https://webaim.org/techniques/css/invisiblecontent/) and doesn't recommend one over the other. The article was last updated in September 2020, so it's probably fine 🤓

Reply:
Great, thanks, seems much better. 

We don't really need a generic solution here, so I've moved the required styles to where they're used.